styles/styles.css

```diff
@@ -72,7 +72,7 @@ body.js-has-sticky-footer {
   }
 }
 
-:root:has(#color-scheme:checked) body {
+body.color-scheme-dark {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
```

Comment:
Any reason we aren't also doing

```css
-webkit-font-smoothing: auto;
-moz-osx-font-smoothing: auto;
```

here? If it's something we missed, let's also update `callout.css` lines 34-35, `feature-layout.css` lines 10-11 (though this can probably be deleted due to lines 69-72), and `styles.css` lines 76-77 and lines 167-168.